config/routes.rb

```diff
@@ -7,6 +7,8 @@
 
   resources :places, only: [:new, :create, :edit, :update]
 
+  get "/feed" => "places#feed", :as => "feed", :defaults => { :format => 'atom' }
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.